src/pages/api/teams/[teamId]/digests/[digestId]/index.ts

```diff
@@ -97,7 +99,15 @@ router
         id: digestId?.toString(),
       },
     });
-
+    const wasAPublishedDigest = Boolean(digest?.publishedAt);
+    if (wasAPublishedDigest) {
+      await client.team.update({
```

Comment:
I think we can probably remove the `await` here since it doesn't impact the digest response.